sdk/servicebus/azure-messaging-servicebus/pom.xml

```diff
@@ -54,7 +54,11 @@
       <artifactId>azure-core-http-netty</artifactId>
       <version>1.9.2</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
-
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
```

Comment:
Same comment as in another PR. Let's not add a new dependency like gson

Reply:
According to your Comment, the added gson dependency is deleted.